main.go

```diff
@@ -304,9 +307,28 @@ func split(s string) (head, tail string) {
 
 func main() {
 	defer task.Exit()
+	ohFlags := flag.NewFlagSet("oh", flag.ContinueOnError)
+
+	interactive := ohFlags.Bool("i", true, "enable interactive mode")
+	command := ohFlags.Bool("c", false, "parses next args as a command to execute")
```

Comment:
This should be a StringVar.

Reply:
Wouldn't that (`-c` being a `StringVar`)  cause the need to quote the whole command leading to a new parsing code instead of using what it's already done?

Reply:
Yes. The whole command needs to be quoted so that another shell doesn't do the parsing. This is how the -c works in other shells too.

Reply:
Should the `-c 'a long script here' ` be treated like a file using a string.Reader and reuse the current file code?

Reply:
Yes, that is what the current code does. See StartNonInteractive in task.go:
```
                b := bufio.NewReader(strings.NewReader(os.Args[2] + "\n"))
                MakeParser(b.ReadString).ParseBuffer(false, "-c", eval)
```